server/routes.js

```diff
@@ -93,7 +93,7 @@ export default (app) => {
    */
   app.post('/users', aN.authenticateAppByApiKey, aZ.appAccess(0.5), required('user'), users.create); // Create a user.
   app.get('/users/:userid', aN.authenticateUserOrApp(), users.show); // Get a user.
-  app.put('/users/:userid', aN.authenticateAppByApiKey, required('user'), aZ.authorizeAccessToUserWithRecentDonation, users.updateUserWithoutLoggedIn); // Update a user.
```

Comment:
if a middleware is only used once, we probably don't need to make it a middleware.
That's why I moved the logic of `aZ.authorizeAccessToUserWithRecentDonation` to the `users` controller (see above)